README.md

```diff
@@ -67,10 +67,21 @@ Launch the application with the following commands:
     ros2 launch hello_world_simulation empty_world.launch.py
     ```
 
+- *Running Simulation Application With Visualization*
+    ```bash
+    export DISPLAY=:0
+    source /usr/share/gazebo/setup.sh
+    source simulation_ws/install/local_setup.sh
+    ros2 launch hello_world_simulation empty_world.launch.py gui:=true
+    ```
+
 Note that when running robot applications on a robot, `use_sim_time` should be set to `false` (which is the default value in `deploy_rotate.launch.py`). When running robot applications along with simulation applications, `use_sim_time` should be set to `true` for both applications (which is the default value in both `rotate.launch.py` and `empty_word.launch.py`).
 
 When running simulation applications, run command with `gui:=true` to run gazebo client for visualization
 
+If the simulation fails with visualization, you can try to set `export DISPLAY=:1`
```

Comment:
i am okay having a general note saying -- "If you fail to see the GUI, try setting `DISPLAY` environment variable to appropriate value. i believe what you have here seems very specific and might not work in all cases.

Reply:
I actually got the DISPLAY=:1 fix from another aws tutorial https://github.com/aws-robotics/aws-robomaker-robotics-curriculum/tree/main/getting-started-with-aws-robomaker. It took me 3 days to figure this out, so I thought it might be helpful to others. 


From the tutorial:

> Switching to the Virtual Desktop tab, you should be able to see the Gazebo GUI with a Turtlebot 3 robot. Note that the command export DISPLAY=:0 wouldn’t be typically necessary on a local machine with Ubuntu. This command is instead necessary on the cloud, as it is not specified which display will be used. At this time, an error might happen preventing Gazebo to run. If that happens, try
`export DISPLAY=:1`